oe_webtools.libraries.yml

```diff
@@ -2,4 +2,4 @@
 drupal.webtools-smartloader:
   version: 1.x
   js:
-   //europa.eu/webtools/load.js: { external: true, attributes: { defer: true } }
+    "https://europa.eu/webtools/load.js": { external: true, attributes: { defer: true } }
```

Comment:
We noticed that `external: true` is wrong, it's actually `type: external` 😅  
Also very nit but the quotes can be avoided in this case.